<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportUnixSSHConfig</class>
 <widget class="QWidget" name="ImportUnixSSHConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>926</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Host: (:port optional)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="HostComboBox">
        <property name="maximumSize">
         <size>
          <width>480</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">combobox-popup:0</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtTop</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="UseSavedCreds">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Use Host-Specific Saved Credentials</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="UseSavedDefaultCreds">
        <property name="text">
         <string>Use Default Saved Credentials</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="UseSpecificCreds">
        <property name="text">
         <string>Use Specific User Credentials</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="CredentialsBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Credentials</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Username:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Username">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="7" column="0">
            <widget class="QRadioButton" name="publicKeyRadio">
             <property name="text">
              <string>Public Key Authentication</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Password:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Password">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="10" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="browsePrivateKeyButton">
               <property name="text">
                <string>Browse...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="8" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Private Key File:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="privateKeyFileEdit"/>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="QRadioButton" name="passwordRadio">
             <property name="text">
              <string>Password Authentication</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QCheckBox" name="SaveDefaultCreds">
             <property name="text">
              <string>Save These Credentials As Default For All Hosts</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QCheckBox" name="SaveCreds">
             <property name="text">
              <string>Save These Credentials For This Host</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Private Key Password:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="privateKeyPassword">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ElevationCredentialsBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Root Credentials</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QRadioButton" name="noElevationRadio">
             <property name="text">
              <string>No Elevation Required</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="sudoRadio">
             <property name="text">
              <string>SUDO</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Password:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="sudoPasswordEdit">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QRadioButton" name="suRadio">
             <property name="text">
              <string>SU</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Password:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="suPasswordEdit">
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="connectAndVerifyButton">
        <property name="text">
         <string>Connect And Verify Credentials</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="credentialsVerifiedLabel">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="text">
         <string>Credentials Verified</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Hash Type To Import:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="hashTypeList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="KeepCurrentAccounts">
        <property name="text">
         <string>Keep Currently Imported Accounts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="IncludeNonLogin">
        <property name="text">
         <string>Include Non-Login And Disabled Accounts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="LimitAccounts">
        <property name="title">
         <string>Limit Number Of Imported Accounts</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Max # of accounts to import:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="AccountLimit">
           <property name="maximumSize">
            <size>
             <width>96</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
           <property name="maxLength">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
